cfgov/unprocessed/css/molecules/image-text-50-50.less

```diff
@@ -82,7 +82,6 @@
 */
 
 .m-image-text-50-50 {
-    padding-bottom: unit( @grid_gutter-width * 2 / @base-font-size-px, em );
```

Comment:
Let me try this in the CMS. I'm pretty sure that it makes you create it two modules at a time, but I'll fix if it allows this and it'll just be something that older IE users will just suffer through.